src/internal/connector/onedrive/collection.go

```diff
@@ -307,16 +308,22 @@ func (oc *Collection) getDriveItemContent(
 	return itemData, nil
 }
 
+type itemGetterAndAugmenter interface {
```

Comment:
Can we add some documentation for this interface?

Reply:
will do